src/core/services/CreditLineService.ts

```diff
@@ -386,14 +389,26 @@ export class CreditLineServiceImpl implements CreditLineService {
     return response;
   }
 
-  // TODO
+  public async getLineEvents(prop: GetLineEventsProps): Promise<GetLineEventsResponse | undefined> {
+    const response = getLineEvents(prop)
+      .then((data) => data)
+      .catch((err) => {
+        console.log('CreditLineService: error fetching line events', err);
+        return undefined;
+      });
+    return response;
+  }
+
   public async getLinePage(prop: GetLinePageProps): Promise<GetLinePageResponse | undefined> {
     const response = getLinePage(prop)
-      .then((data) => data)
+      .then((data) => {
```

Comment:
y this change?

Reply:
I was debugging something - switched this back to the way it was